DigitalLearningSolutions.Web/Views/TrackingSystem/Centre/Administrator/DeactivateAdminConfirmation.cshtml

```diff
@@ -0,0 +1,20 @@
+﻿@{
+  ViewData["Title"] = "User deactivated";
```

Comment:
We are also going to want the other ViewData set here as it is in the other page, so it should all look like this:
```
ViewData["Title"] = "User deactivated";
ViewData["Application"] = "Tracking System";
ViewData["HeaderPath"] = $"{Configuration["AppRootPath"]}/TrackingSystem/Centre/Dashboard";
ViewData["HeaderPathName"] = "Tracking System";
```
The ViewData just sets up a few key bits of information that is used in the layout so make sure the DLS logo is correct, so its important to have these correct on each page.

Reply:
Added the required properties to view data